common/src/main/java/com/wynntils/features/utilities/PerCharacterGuildContributionFeature.java

```diff
@@ -73,6 +78,30 @@ public void onChatReceived(ChatMessageReceivedEvent event) {
 
             characterContributions.get().put(Models.Character.getId(), contributionAmount);
             characterContributions.touched();
+
+            sendContributionMessage(contributionAmount);
         }
     }
+
+    private void sendContributionMessage(int contributionAmount) {
+        MutableComponent contributionMessage = Component.literal(ChatFormatting.DARK_AQUA + "You will now contribute "
```

Comment:
This part is the exact same as the default message but the message we send appears before the default one so I decided to cancel the default and just add this here, but would it be better to not cancel it and just remove this first part and delay sending the message by a few ticks?

Reply:
No, that is worse I think. This is fine.